docs/REST API/rest-api.md

```diff
@@ -4175,7 +4177,13 @@ Resends a configuration profile for the specified host.
           "version": "121.0",
           "last_opened_at": "2024-04-01T23:03:07Z",
           "vulnerabilities": ["CVE-2023-1234","CVE-2023-4321","CVE-2023-7654"],
-          "installed_paths": ["/Applications/Google Chrome.app"]
+          "installed_paths": ["/Applications/Google Chrome.app"],
+          "signature_information": [
```

Comment:
Should we rename this considering we may re-use this field in the future for Linux/Windows related information around the paths?

Alternatives: "paths", "paths_info", I'm all ears :)

Reply:
Nevermind, `signature_information` will only be returned for apps. In the future we can return other sub-keys for Windows/Linux if need be.